api/managementpb/actions.proto

```diff
@@ -105,6 +106,24 @@ message StartMySQLExplainActionResponse {
   string pmm_agent_id = 2;
 }
 
+message StartMySQLJsonExplainActionRequest {
```

Comment:
StartMySQLExplainJSONActionRequest. Common prefix "StartMySQLExplain" for grouping, JSON for acronym.